pkgs/top-level/all-packages.nix

```diff
@@ -1550,6 +1550,8 @@ in
 
   libbladeRF = callPackage ../development/libraries/libbladeRF { };
 
+  libgpuarray-cuda = python34Packages.libgpuarray-cuda;
+
```

Comment:
`libgpuarray` is supposed to be a `python` package. You are right, it is best not to hard-wire a particular choice of the version of `pythonPackages` and to delete this definition.